lib/command/base.rb

```diff
@@ -294,6 +352,30 @@ def wait_for_replica(workload, location)
       end
     end
 
+    def start_cron_workload(workload, location)
+      step("Starting workload", retry_on_failure: true) do
+        cp.workload_cron_start(workload, location: location)
+      end
+    end
+
+    def wait_for_cron_workload_status(workload, location) # rubocop:disable Metrics/MethodLength
+      loop do
+        return 0 if cp.fetch_workload(workload).nil?
+
+        result = cp.workload_cron_get(workload, location: location)
+        status = result&.dig("items", 0)&.dig("status")
+
+        case status
+        when :invalid, :removed, :failure
+          return 1
+        when :success
+          return 0
+        end
+
+        Kernel.sleep(1)
```

Comment:
By default cron jobs run indefinitely. Do we want to add a forced timeout or add an option to specify timeout?

If we add a timeout to the cron job, we don't need to add a timeout here, because this loop finishes once the job no longer exists.